src/org/labkey/test/pages/admin/RenameFolderPage.java

```diff
@@ -0,0 +1,84 @@
+package org.labkey.test.pages.admin;
+
+import org.labkey.test.Locator;
+import org.labkey.test.WebDriverWrapper;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.pages.LabKeyPage;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class RenameFolderPage extends LabKeyPage<RenameFolderPage.ElementCache>
+{
+    public RenameFolderPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public static ReorderFoldersPage beginAt(WebDriverWrapper driver, String containerPath)
```

Comment:
Why is the beginArt for the the **Rename**FolderPage returning a **Reorder**FolderPage?

Reply:
good catch